covfee/client/admin/hit_block/journey_buttons.tsx

```diff
@@ -47,16 +73,22 @@ export const JourneyRow = ({
                 journey.num_connections == 0
                   ? "gray"
                   : journey.num_connections == 1
-                    ? "green"
-                    : "red",
+                  ? "green"
+                  : "red",
             }}
           >
             <ApiOutlined />
           </span>
         </LinkContainer>
         <span> </span>
-        <span>{journey.id.substring(0, 10)}</span> <LinkOutlined />
+        <span>{journey.id.substring(0, 10)} </span> <LinkOutlined />
       </a>
+      {show_annotator_data && (
```

Comment:
I think you can remove the state of `show_annotator_data` and simply use `annotator_data && ...` as they are redundant, i.e., show_annotator_data is true always when annotator_data is not null

Reply:
Neat! Fixed in 06bbf53eba006fb394fef90cc275f8062d7a3762